apps/web/src/app/blog/page.tsx

```diff
@@ -49,54 +63,77 @@ export default function BlogPage({ searchParams }: { searchParams: { tag?: strin
         <FilterSelectBox selectedTag={selectedTag} blogTags={blogTags} />
 
         <ul className="blog-posts-list">
-          {paginatedBlogs.map((post, index) => (
-            <li
-              key={index}
-              className="blog-post-item active"
-              data-category={post.metadata.category}
-            >
-              <Link href={`/blog/${post.slug}`} rel="noopener noreferrer">
-                <figure className="blog-banner-box">
-                  <Image
-                    src={post.metadata.banner}
-                    alt={post.metadata.alt || 'Blog post image'}
-                    loading="lazy"
-                    width={1600}
-                    height={900}
-                  />
-                </figure>
-                <div className="blog-content">
-                  <div className="blog-meta">
-                    <p className="blog-category">{post.metadata.category}</p>
-                    <span className="dot"></span>
-                    <time dateTime={post.metadata.publishedAt}>
-                      {new Date(post.metadata.publishedAt).toLocaleDateString(
-                        'en-us', { month: 'long', day: 'numeric', year: 'numeric' }
-                      )}
-                    </time>
+          {isLoading
+            ? Array.from({ length: POSTS_PER_PAGE }).map((_, index) => (
+                <li key={index} className="blog-post-item skeleton">
+                  <Skeleton height={200} className="blog-banner-skeleton" />
```

Comment:
Please use `SkeletonTheme` with baseColor, `#202020`, and highlightColor, `#444`

```tsx
<SkeletonTheme baseColor="#202020" highlightColor="#444">
  <p>
    <Skeleton count={3} />
  </p>
</SkeletonTheme>
```
```